packages/specification/schemas/components/MedicationRequest/ReleaseMedicationRequest.yaml

```diff
@@ -161,10 +161,12 @@ properties:
           Duration of each issue in days; defaults to 28.
         properties:
           value:
-            type: integer
-            description: The number of days each issue in an eRD prescription is valid for.
-            format: int32
-            default: 28
+            type: number
```

Comment:
This should not change. The expectedSupplyDuration should just be an integer as you can't have something valid for 28.5 days.